lib/generators/frontier_model/templates/model.rb

```diff
@@ -4,6 +4,12 @@ class <%= model_configuration.as_constant %> < ActiveRecord::Base
   acts_as_paranoid
 
 <% # The whitespace here is very important, we only want to include a blank line if there are some enums, etc -%>
+<% if model_configuration.attributes.flat_map(&:constants).any? -%>
+<% model_configuration.attributes.flat_map(&:constants).each do |constant| -%>
+  <%= constant.model_implementation %>
+<% end -%>
```

Comment:
I'm obviously missing something important. Why do you need the if statement on line 7? Won't the each statement on line 8 get ignored if there arn't any?